From 4ddbf92576f967628ba116b7a1d4500e3ed15df0 Mon Sep 17 00:00:00 2001
From: LiangRongrong <Liangrongrong@mprc.pku.edu.cn>
Date: Tue, 29 Nov 2011 21:40:26 +0800
Subject: [PATCH 098/641] UniCore64:fix checkpatch errors in asm dma-mapping.h

ERROR: Macros with complex values should be enclosed in parenthesis
       get_dma_ops(dev)->dma_supported((d), (m))

Signed-off-by: Liang Rongrong <liangrongrong@mprc.pku.edu.cn>
---
 arch/unicore64/include/asm/dma-mapping.h |    6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/arch/unicore64/include/asm/dma-mapping.h b/arch/unicore64/include/asm/dma-mapping.h
index 9c408bf..b8f7f4d 100644
--- a/arch/unicore64/include/asm/dma-mapping.h
+++ b/arch/unicore64/include/asm/dma-mapping.h
@@ -6,11 +6,11 @@ extern struct dma_map_ops swiotlb_dma_map_ops;
 #define get_dma_ops(dev)		(&swiotlb_dma_map_ops)
 
 #define dma_supported(d, m)			\
-		get_dma_ops(dev)->dma_supported((d), (m))
+		(get_dma_ops(dev)->dma_supported((d), (m)))
 #define dma_alloc_coherent(d, s, h, f)		\
-		get_dma_ops(dev)->alloc_coherent((d), (s), (h), (f))
+		(get_dma_ops(dev)->alloc_coherent((d), (s), (h), (f)))
 #define dma_free_coherent(d, s, a, h)		\
-		get_dma_ops(dev)->free_coherent((d), (s), (a), (h))
+		(get_dma_ops(dev)->free_coherent((d), (s), (a), (h)))
 
 #define dma_alloc_noncoherent(d, s, h, f)	\
 		dma_alloc_coherent((d), (s), (h), (f))
-- 
1.7.9.5

